web-api/src/business/useCases/practitioner/updatePractitionerUserInteractor.ts

```diff
@@ -149,29 +155,6 @@ export const updatePractitionerUser = async (
   }
 };
 
-const updateUserPendingEmail = async ({
```

Comment:
This function was very similar to behavior occurring in `updateUserPendingEmailInteractor`. I refactored the function slightly and put it in `updateUserPendingEmailInteractor`, where it seems a more natural fit.